src/components/Map.js

```diff
@@ -1,26 +1,26 @@
-import React from 'react';
-import GoogleMapReact from 'google-map-react';
+import React from "react";
+import PropTypes from "prop-types";
+import GoogleMapReact from "google-map-react";
+import MyLocation from "./MyLocation";
 
-function Map(){
-    const defaultLocation = {
-        center:{
-            lat: 59.95,
-            lng: 30.33
-        },
-        zoom: 11
+function Map(props) {
+  const { data } = props;
```

Comment:
```suggestion
function Map({data}) {
```
This simplifies the code by unpacking the props and assigning to a variable within the parameter.

Reply:
👍

Reply:
Smooth. I like it. :-)